src/Modules/Windows/PSDiagnostics/PSDiagnostics.psm1

```diff
@@ -437,12 +437,12 @@ namespace Microsoft.PowerShell.Diagnostics
 }
 "@
 
-if ($psedition -eq 'Core')
- {
-    # Currently we only support these cmdlets as logman.exe is not working on Nano/Lot system.
-    Export-ModuleMember Enable-PSTrace, Disable-PSTrace, Get-LogProperties, Set-LogProperties
- }
- else
- {
-    Export-ModuleMember Start-Trace, Stop-Trace, Enable-WSManTrace, Disable-WSManTrace, Enable-PSTrace, Disable-PSTrace, Enable-PSWSManCombinedTrace, Disable-PSWSManCombinedTrace, Get-LogProperties, Set-LogProperties
- }
+if (Get-Command logman.exe -Type Application -ErrorAction SilentlyContinue)
```

Comment:
Seems a comment about Nano/IoT will be still useful.

Reply:
The comment still exists on line 446